pkg/apis/storage/validation/validation.go

```diff
@@ -44,8 +44,8 @@ const (
 	maxAttachedVolumeMetadataSize = 256 * (1 << 10) // 256 kB
 	maxVolumeErrorMessageSize     = 1024
 
-	csiNodeIDMaxLength       = 128
-	csiNodeIDLongerMaxLength = 192
+	csiNodeIDMaxLength       = 192
```

Comment:
this would not work well, if someone rollback from a highly versions

Reply:
This is to keep the CSINode ID length backwards compatible.  I dont think we need to support the other way around.